databricks-iris/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/hooks.py

```diff
@@ -0,0 +1,24 @@
+from kedro.framework.hooks import hook_impl
+from pyspark import SparkConf
+from pyspark.sql import SparkSession
+
+
+class SparkHooks:
+    @hook_impl
+    def after_context_created(self, context) -> None:
+        """Initialises a SparkSession using the config
+        defined in project's conf folder.
+        """
+
+        # Load the spark configuration in spark.yaml using the config loader
+        parameters = context.config_loader.get("spark*", "spark*/**")
+        spark_conf = SparkConf().setAll(parameters.items())
```

Comment:
probably we should update it, we should use the newer method `context.config_loader["spark"]` instead

Reply:
Good note, I'd keep it separate from the scope of this PR as it impacts `pyspark-iris` also.